rest-api/src/main/java/com/cloudbees/workflow/rest/external/ChangeSetExt.java

```diff
@@ -173,7 +181,7 @@ protected void mapFields(ChangeLogSet<? extends ChangeLogSet.Entry> changeset, W
             commit.setCommitId(entry.getCommitId());
             commit.setCommitUrl(repoUrl);
             commit.setMessage(entry.getMsg());
-            commit.setAuthorJenkinsId(entry.getAuthor().getFullName());
+            commit.setAuthorJenkinsId(isResolvingCommitAuthors() ? entry.getAuthor().getFullName() : "Unidentified");
```

Comment:
Red Hat's Openshift Team is directly consuming some of our APIs, see https://github.com/jenkinsci/pipeline-stage-view-plugin/pull/11#issuecomment-217139203